.github/workflows/tests.yaml

```diff
@@ -46,10 +46,10 @@ jobs:
 
     strategy:
       matrix:
-        php-version: ['5.6', '7.1', '7.2', '7.3', '7.4', '8.0']
+        php-version: ['7.3', '7.4', '8.0']
         dependencies: ['']
         include:
-          - { php-version: '5.6', dependencies: '--prefer-lowest' }
+          - { php-version: '7.3', dependencies: '--prefer-lowest' }
```

Comment:
In that case we would for example need to keep compatibility with older PHPUnits (see https://phpunit.de/supported-versions.html )...

Reply:
That would be okay as the signatures of the methods would not be an issue.
For example phpMyAdmin is running from PHP 7.2 to 8.2 on 5.2.x and was from 7.1 to 8.2 on 5.1.x.
So no big deal

Reply:
For clarity I am asking for 7.2 because we will need support for 7.2 at phpMyAdmin until 5.2.x goes out of support

Reply:
> That would be okay as the signatures of the methods would not be an issue.

There are alsi PHPUnit features which were added in PHPUnit 9 and even 9.3. So using PHPUnit 8 would not allow using them. Also for the migrated configuration - to support both PHPUnit 8 and (future) 10, you would have to have different configurations as a workaround, because they are incompatible. Also some methods were deprecated in PHPUnit 9, their replacement were added also in 9.x and will be removed in 10.x So this brings some additional complexity, which I'd prefer to avoid.

Reply:
I'm okay with with this bump. They don't having to support PHPUnit 8 will decrease more of their maintenance work than will increase ours.